src/sass/_Fabric.Typography.scss

```diff
@@ -344,8 +344,12 @@
   color: $ms-color-success;
 }
 
-@mixin ms-fontColor-alert {
-  color: $ms-color-alert;
+@mixin ms-fontColor-warning {
```

Comment:
The documentation will also need to be updated to include the 'warning' and 'severeWarning' colors.